src/pages/CityPage/CityPageGuides.jsx

```diff
@@ -0,0 +1,141 @@
+import { useState, memo } from 'react';
+import { Link } from 'react-router-dom';
+
+import GuideCard from '../../components/common/GuideCard/GuideCard';
+import CategoryCard from '../../components/common/CategoriesCards/CategoryCard';
+import CreateGuideButton from '../../components/common/CreateGuideButton/CreateGuideButton';
+
+import HistoricalIcon from '../../assets/img/CategoriesCards/Icons/HistoricalIcon.svg';
+import PhotoIcon from '../../assets/img/CategoriesCards/Icons/PhotoIcon.svg';
+import FoodIcon from '../../assets/img/CategoriesCards/Icons/FoodIcon.svg';
+import CultureIcon from '../../assets/img/CategoriesCards/Icons/CultureIcon.svg';
+import ParksIcon from '../../assets/img/CategoriesCards/Icons/ParksIcon.svg';
+import MarketIcon from '../../assets/img/CategoriesCards/Icons/MarketIcon.svg';
+
+import HistoricalLocations from '../../assets/img/CategoriesCards/HistoricalLocations.png';
+import PhotoLocations from '../../assets/img/CategoriesCards/PhotoLocations.png';
+import FoodTour from '../../assets/img/CategoriesCards/FoodTour.png';
+import CulturalEvents from '../../assets/img/CategoriesCards/CulturalEvents.png';
+import Parks from '../../assets/img/CategoriesCards/Parks.png';
+import Markets from '../../assets/img/CategoriesCards/Markets.png';
+import NoCategory from '../../assets/img/CategoriesCards/NoCategory.png';
+import Stub from '../../assets/img/cities/Stub.png';
+
+const CategoriesButtonsData = [
+  {
+    categoryName: 'Historical landmarks',
+    icon: <HistoricalIcon />,
+    backgroundImage: HistoricalLocations,
+  },
+  {
+    categoryName: 'Photographic locations',
+    icon: <PhotoIcon />,
+    backgroundImage: PhotoLocations,
+  },
+  {
+    categoryName: 'Food Tour',
+    icon: <FoodIcon />,
+    backgroundImage: FoodTour,
+  },
+  {
+    categoryName: 'Cultural events and museums',
+    icon: <CultureIcon />,
+    backgroundImage: CulturalEvents,
+  },
+  {
+    categoryName: 'Parks and natural areas',
+    icon: <ParksIcon />,
+    backgroundImage: Parks,
+  },
+  {
+    categoryName: 'Traditional markets and souvenir shops',
+    icon: <MarketIcon />,
+    backgroundImage: Markets,
+  },
+  {
+    categoryName: 'Without Category',
+    backgroundImage: NoCategory,
+  },
+];
+
+const CityPageGuides = () => {
+  const [activeButtons, setActiveButtons] = useState([]);
+
+  const handleButtonClick = buttonName => {
+    setActiveButtons(prevActiveButtons => {
+      if (buttonName === 'Without Category') {
+        return prevActiveButtons.includes(buttonName)
+          ? []
+          : CategoriesButtonsData.map(button => button.text);
```

Comment:
It reminds me PR for Travel Tips. Let's not to rely on button text which changes often. It's better to make `key` field and store the unique value there.